games/strikers/src/client/scene-manager.tsx

```diff
@@ -70,16 +82,19 @@ const GameScene = () => {
   const cells = Array.from(grid).map((cell) => {
     return cell;
   });
-  const { gameEntity } = useContext(StrikersContext);
-  console.log({ gameEntity });
+  const { gameEntity, strikersPlayerEntity } = useContext(StrikersContext);
+  console.log({ gameEntity, strikersPlayerEntity });
   const players = useEntitySelector(
     gameEntity,
     (entity) => entity.gameState.players
   );
+  const playerCameraPosition = useEntitySelector(
```

Comment:
doing this is going to have the rig provider re-render frequently... instead what you can do is just access the entity from within the CameraRigProvider using StrikersContext, and then to strikersPlayerEntity.subscribe from within a `useEffect` and then

and then you can do `const { camera } = useThree()` to get the camera to call it from within that subscribe call